### ISSUE TITLE
 x = torch.cat((x1_downSample, x2_downSample, x3), dim=1)

### ISSUE THREAD

**Issue by longmalongma** [MingyeXu/GS-Net#6]
x = torch.cat((x1_downSample, x2_downSample, x3), dim=1)
麻烦请教一下点数不一样怎么进行融合？
